File: python/Repetitions.py

st = input()

current_char = 'D'
count = 0
ans = 0

for i in range(len(st)):
    if st[i] != current_char:
        ans = max(ans, count)
        count = 0
    
    count += 1
    current_char = st[i]
ans = max(ans, count)
print(ans)

File: cpp/Repetitions.cpp
#include<bits/stdc++.h>
using namespace std;

#define  ll long long


int main(){
	string st; cin >> st;

	char prevous_char = 'D';
	int ans = 0, current_mx = 0;

	for (int i = 0; i < st.size(); i ++){
		if(st[i] != prevous_char){
			ans = max(ans, current_mx);
			current_mx = 0;
		}

		current_mx ++;
		prevous_char = st[i];
	}
	ans = max(ans, current_mx);
	cout << ans << "\n";
}
